client/packages/common/src/intl/locales/tet/replenishment.json

```diff
@@ -0,0 +1,102 @@
+  "label.cant-delete-disabled": "Bele deit hamos linas wainhira nia estatus foun",
+  "label.soh": "SOH",
+  "label.stock-level": "Nivel Stock",
+  "label.stock-on-hand": "Stock on Hand",
+  "label.suggested-order-quantity": "Kuantidade pedidu ne'ebe sujere",
+  "label.suggested-quantity": "Kuantidade ne'ebe sujere",
+  "label.supplier-name": "Naran Distribuidór",
+  "label.supplier-ref": "Referensia",
+  "label.shipment": "Remessa",
+  "messages.no-shipments-yet": "Seidauk iha Kriasaun ba Remessas Nian.",
+  "messages.by-user": "Ho {{username}}",
+  "label.add-charges": "Aumenta Cobrancas",
+  "messages.requested-to-suggested": "Ne'e sei regula linas pedidu hotu ba iha kuantidade ne'ebe sujere.",
+  "messages.no-related-documents": "Laiha dokumentus relevantes",
+  "messages.inbound-shipment-created-on": "Remessa sasan tama kria iha {{date}}",
+  "messages.off-hold-confirmation": "Ne'e sei reativa fali mudansa ba status",
+  "messages.on-hold-confirmation": "Ne'e sei prevene mudansa ba estatus to'o \"hold\" ne'e Desativa tiha ona",
+  "messages.internal-order-created-on": "Pedidu Internal kria iha {{date}}",
+  "messages.cant-delete-requisitions": "Bele deit hamos requizisaun ne'ebe ho estatus 'Draf'",
+  "messages.click-to-return-to-shipments": "Labele atu identifika remessa ho ID ida ne'e. Klik OK atu fila fali ba lista remessa",
+  "messages.deleted-shipments_one": "Hamos ona {{count}} Remessa",
+  "messages.deleted-shipments_other": "Hamos ona {{count}} Remessas",
+  "messages.confirm-delete-shipments_one": "Ne'e sei hasai permanente remessa 1",
+  "messages.error-saving-shipment": "Iha erru atu salva remessa🥺",
+  "messages.select-rows-to-delete-them": "Hili lina sira atu hamos",
+  "messages.shipment-saved": "Remessa salva ona🥳",
+  "label.min": "Minimu",
+  "messages.confirm-changing-from-new": "Notas: Muda husi 'foun' sei hamos lina ne'ebe ho kuantidade Zero.",
+  "messages.confirm-inbound-status-as": "Konfirma estatus nudar $t({{status}})?",
+  "messages.could-not-save": "Labele atu salva",
+  "messages.confirm-delete-invoice-lines_one": "Ne'e sei hasai permanente lina 1 husi Invoice ida ne'e",
+  "messages.confirm-delete-requisition-lines_one": "Ne'e sei hasai permanente lina 1 husi pedidu ida ne'e",
+  "messages.confirm-delete-requisition-lines_other": "Ne'e sei hasai permanente {{count}} linas husi pedidu ida ne'e",
+  "messages.confirm-delete-requisitions_one": "Ne'e sei hasai permanente pedidu internal 1",
+  "messages.confirm-delete-requisitions_other": "Ne'e sei hasai permanente {{count}} pedidu internal",
+  "messages.confirm-delete-shipment-lines_other": "Ne'e sei hasai permanente {{count}} linas remessa",
+  "messages.deleted-requisitions_one": "Hamos ona {{count}} pedidu",
+  "messages.deleted-requisitions_other": "Hamos ona {{count}} pedidu sira",
+  "messages.confirm-status-as": "Konfirma estatus hanesan $t({{status}})?",
+  "messages.saved": "Salva ona",
+  "label.rnr-adjustments": "Ajustamentu",
```

Comment:
I think we need to move these to common now for `develop`?

Reply:
Oh right nice catch James XD